internal/gps/pkgtree/pkgtree_test.go

```diff
@@ -19,6 +19,7 @@ import (
 
 	"github.com/golang/dep/internal/fs"
 	"github.com/golang/dep/internal/gps/paths"
+	_ "github.com/golang/dep/internal/test" // Define test flags globally
```

Comment:
ack ack ack...really don't want to breaking gps' encapsulation to solve this problem :(